lib/core/src/msParam.cpp

```diff
@@ -1022,6 +1022,34 @@ parseMspForPosInt( msParam_t* inpParam ) {
     return myInt;
 }
 
+int msp_to_string(msParam_t* inp_param, char** out_param)
+{
+    if (out_param == nullptr) {
```

Comment:
Consider swapping the order of the operands to avoid accidental writes.

_This applies to other conditionals too._